members.yaml

```diff
@@ -15,3 +15,10 @@
   lastName: Lewi
   affiliation: Google
   email: jlewi@google.com
+
+- github: ddutta
+  slack: debo
+  firstName: Debo
+  lastName: Dutta
+  affiliation: Cisco
+  email: dedutta@cisco.com
```

Comment:
I think it has conflicts with your another PR #67 , maybe we could remove the changes here.

Reply:
thx ..

Reply:
I committed when the branch did have the conflict. Will resolve it ASAP